src/handler/job/monitorRanks.ts

```diff
@@ -1,17 +1,36 @@
-import { EventHandler, ScheduledJob, Cron } from "../../decorators";
-import { Client } from "discord.js";
+import {
+    EventHandler,
+    ScheduledJob,
+    Cron,
+    DiscordEvent,
+} from "../../decorators";
+import { Client, GuildMember, PartialGuildMember, Guild } from "discord.js";
 
 import { UserRankRepository } from "../../repository/UserRankRepository";
+import { AppDataSource } from "../..";
 
-import { processUser } from "../../service/nickname";
+import { processUser } from "../../service/verifyService";
 import { makeLogger } from "../../logger";
+import { UserRank } from "../../entity/UserRank";
+import { removeMembers } from "../../service/nicknameService";
+import { isSome } from "../../util/guard";
+import { getUser } from "../../wrapper/wrapper";
+import { config } from "../../config";
+import { ApiError } from "../../error";
+
+import {
+    getGuild,
+    sendMessage,
+    updateRoleAndNickname,
+} from "../../service/verifyService";
+import { Color } from "../../service/util/constants";
 
 const logger = makeLogger(module);
 
 @EventHandler()
 @ScheduledJob()
 export class MonitorRanks {
-    @Cron("*/10 * * * *")
+    @Cron("* * * * *")
```

Comment:
Is this intentional?

Reply:
this is outdated, already fixed. was an accident